json-rpc/src/util.rs

```diff
@@ -110,8 +116,10 @@ impl SdkInfo {
         let lowercase_user_agent = user_agent.to_lowercase();
         let user_agent_parts: Vec<&str> = lowercase_user_agent.split('/').collect();
         if user_agent_parts.len() == 2 {
-            let language = SdkLang::from_str(&user_agent_parts[0]);
-            let version = SdkVersion::from_str(&user_agent_parts[1]);
+            let language =
+                SdkLang::from_str(&user_agent_parts[0]).expect("could not parse sdk language");
+            let version =
+                SdkVersion::from_str(&user_agent_parts[1]).expect("could not parse sdk version");
```

Comment:
I feel this change makes code worse, as it is confusing whether the expect will cause panic

Reply:
this was forced via clippy wanting `from_str` to use the `FromStr` trait. I'm open to suggestions

Reply:
Rename to SdkLang::parse?